rs/bitcoin/ckbtc/minter/src/state.rs

```diff
@@ -382,7 +379,7 @@ pub struct CkBtcMinterState {
     pub owed_kyt_amount: BTreeMap<Principal, u64>,
 
     /// A cache of UTXO KYT check statuses.
-    pub checked_utxos: BTreeMap<Utxo, (String, UtxoCheckStatus, Principal)>,
+    pub checked_utxos: BTreeMap<Utxo, (Option<String>, UtxoCheckStatus, Option<Principal>)>,
```

Comment:
nit: The type of the values is a bit hard to understand, since the semantic is not apparent from the type (e.g. `String` is actually meant to represent a UUID to identify a provider). Maybe it would be better to have this as a proper `struct`? If yes, also ok obviously to do this in a separate PR.